opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -452,6 +491,10 @@ message HistogramDataPoint {
   // If we decide to also support (a, b] intervals we should add support for these by defining
   // a boolean value which decides what type of intervals to use.
   repeated double explicit_bounds = 7;
+
+  // (Optional) List of exemplars collected from 
+  // measurements that were used to form the data point
+  repeated RawValue exemplars = 8;
```

Comment:
Can you have more than one exemplar per bucket, similar to how you made the exemplars repeated for int/double? If so the indices wouldn't line up anymore with `bucket_counts`.

Idk the performance implications, but you could also have a `message Bucket` with the counts + bounds + exemplar for that bucket?

Reply:
Exemplars are not sent as per-bucket - that can be determined afterwards if necessary. You can have multiple exemplars per bucket and they will all be combined here - indices not relevant.